src/drawing/__tests__/drawingUtil-test.ts

```diff
@@ -0,0 +1,35 @@
+import { Annotation } from '../../@types';
+import { annotations } from '../__mocks__/data';
+import { getCenter, getShape, isDrawing } from '../drawingUtil';
+
+describe('drawingUtil', () => {
+    describe('getCenter()', () => {
+        test('should return the position of the center of the shape', () => {
+            expect(getCenter({ height: 50, width: 50, x: 0, y: 0 })).toMatchObject({ x: 25, y: 25 });
```

Comment:
Should we pass in some unusual values, like `{ height: 0, width: 0, x: 0, y: 0 }` to ensure things work as expected?